### DISCUSSION TITLE
the services of kubernetes  has no data in  dashboard at v9.4

### DISCUSSION THREAD

**Discussion by ronnie-techsun** [apache/skywalking#10650] [General]
1.the kube-stat-metrics exporter and data were ready.
2.the OpenTelemetry receiver was config reference the example  from offical.

my question is whey does no data shown at services of kubernetes  in dashboard? anything else miss? 

thanks..

<img width="1418" alt="image" src="https://user-images.githubusercontent.com/130015511/230251138-daf5367a-9238-476e-a2e2-6870c7331d77.png">

**Comment by ronnie-techsun** [comment]
the OpenTelemetry receiver configured as follow:

![img_v2_7d67fe75-fcdc-46ac-b622-2456c95319bg](https://user-images.githubusercontent.com/130015511/230251866-c06c9641-adee-4f4b-9909-8c6457a6e258.jpg)

**Comment by wu-sheng** [comment]
Try showcase, it is out of box for demo use.
